### PR TITLE
Added part 2 of hype detector blog post

### PR DESCRIPTION
**Pull request by pdamodaran**
This is part 2 of a blog post series that prompts organizations to think critically when considering using LLMs in their AI projects.

### CODE REVIEW

File: content/en/blog/2024-05-08-to-llm-or-not-to-llm-that-is-the-question-part-2.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: "To LLM or Not to LLM, That Is the Question (Part 2)"
+slug: "to-llm-or-not-to-llm-that-is-the-question-part-2"
+date: "2024-05-09T09:00:00-05:00"
```
Comment (daniellemaxwell) [inline content/en/blog/2024-05-08-to-llm-or-not-to-llm-that-is-the-question-part-2.md:4]: The date will be changed to release a week after part 1 is published.

File: content/en/blog/2024-05-08-to-llm-or-not-to-llm-that-is-the-question-part-2.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+authors: ['Danielle Maxwell', 'Prema Roman']
+profile: img/butterfly.png
+tags: ['LLMs', 'AI', 'ML', 'Python', 'Data']
+description: "In part 2 of our series, we examine whether using simpler ML models could be the solution"
+---
+What if you could check whether the content you've read on a social media site is genuine or full of hot air?
+
+<!--more-->
+
+In the last post, we discussed that while LLMs have unlocked new use cases that companies can leverage to generate more revenue, they are not the catch-all solution for every single AI use case.  We identified the following drawbacks:
```
Comment (daniellemaxwell) [inline content/en/blog/2024-05-08-to-llm-or-not-to-llm-that-is-the-question-part-2.md:17]: We'll add a link to the 1st post once it's published.

**Review by rebeccabilbro — COMMENTED** [review]
Hey @pdamodaran @daniellemaxwell apologies for the delay -- just a heads up that I am working on this one too, about halfway through. I've pushed up edits from my first pass, and will push up the rest by the end of the week so we can publish on Monday 20th!

**Review by rebeccabilbro — APPROVED** [review]
Another great post!

**Pull request merged**